### PR TITLE
part 3

### PR DESCRIPTION
**Pull request by shaqsingh**

### CODE REVIEW

**Comment by suchthis** [comment]
Thanks so much for all this work @shaqsingh! It looks awesome. I scanned through and here are all the things I could find; some may have already been decided by you and @dcwalk and I missed it; sorry, if so!


* The last link in the ToC ("Digital Information Governance Practices") goes to the last comment rather than to that section

* We're missing the link from "EDGI's Recommendations" in the ES to the "Digital Information Governance Practices" section
 
* The figures are in the wrong order and some are missing

* I think we should add margin and make sure it's consistent above and below figures/captions

* The ToC formatting seems off (from both PDF version and previous reports). Can we use the spacing / roman numeral sizes / font sizes from PDF? It seems a bit odd to me to use a smaller font size in the ToC than our default body copy size, and roman numerals smaller than font size, etc.

* The font size gets smaller in ULs (issue exists in prev report pages, too)

* The author line gets cut off / not appropriately scaled to screen size for certain window widths (e.g. the one I happened to have in Safari, OSX, MBP) (Screenshot attached)

authors-cut-off

* I think bolding the "Figure X" in the figure caption helps set it off from the text - did we change this for a reason?

* The link styling is different from last report, where we used underline. I think we should be consistent across the 100Days site, and if we're going to remove the underline, should do for all Parts, but should prob be a group decision, as I know we discussed this at length last time and group decision was to use underline. If we change to no underline, maybe we should match style of main site, with transition effect?

* We need to add nav items for Data Methods & Data Visualization (Attached is a possible icon for the latter - haven't tried it in place but looks like it would match the others and not be too similar to Data Methods)



* Should we make external links open in new tab, so we're not taking ppl away frmo the report? (as we do on main EDGI site)

* Should we have nav element linking back to main EDGI site? It's pretty hard to find one's way to the main site from 100Days.

* Did we intentionally add the date to the top of this report? We didn't use in other reports, and I personally think it looks a bit better without (seems to make it feel instantly dated somehow)

* We are still using different heading/subheading styling than we established early on for our report template and used in the previous report PDFs (I think we discussed this during discussion of prev web versions, but don't remember if it was resolved) do we want to be consistent w/ heading/subheading style or diverge online?

* Can we match the section break in section "MISUSE OF LANGUAGE AND MISUNDERSTANDINGS" with that used in the PDF? Seems to stand out better when it's more spacious and uses more asterisks. Would be great to get same spacing above/below it, but I know sometimes spacing is tricky online.

* The comment boxes are different from PDF in that: a) heading style uses all caps, smaller font, and no horizontal rule, b) there's less margin above/below boxes and less padding within, and c) font color is black instead of charcoal grey. Were all those changes intentional?

* I think it'd be nice to add a little space at the end of the report before EDGI logo and make space above/below the HR consistent

* Should we increase the Footnotes/Comments headings to H2? They look a little small.

* The image on the EPA Under Siege page isn't correctly scaling or isn't the correct image; should match the image on the index page

* Should we consider adjusting the Title/Subtitle size on individual report pages? The title seems really big and subtitle pretty small. 

* We still need to resolve question of using the cover image used for PDF vs the after-image in use now

**Comment by trinberg** [comment]
- The last link in the ToC ("Digital Information Governance Practices") goes to the last comment rather than to that section
> Good catch - this should be changed.

- We're missing the link from "EDGI's Recommendations" in the ES to the "Digital Information Governance Practices" section
> +1, should be added

- The figures are in the wrong order and some are missing
> yes, all the figures are wrong / missing. Let me know how I can help with this. You can either use the google doc as reference or refer to this document for links to the figures: https://docs.google.com/document/d/1aO2KjYM8KgdxvUjVjOyTvOZTy3j6NogFWXJkSN7EhVU/edit

- I think we should add margin and make sure it's consistent above and below figures/captions
> Agreed, but will follow your lead on the actual spacing 

- The ToC formatting seems off (from both PDF version and previous reports). Can we use the spacing / roman numeral sizes / font sizes from PDF? It seems a bit odd to me to use a smaller font size in the ToC than our default body copy size, and roman numerals smaller than font size, etc.
> +1, let's normalize

- The font size gets smaller in ULs (issue exists in prev report pages, too)
> not seeing what this refers to, but agree that font size should stay consistent.

- The author line gets cut off / not appropriately scaled to screen size for certain window widths (e.g. the one I happened to have in Safari, OSX, MBP) (Screenshot attached)
> Yes, we should definitely include all the authors :)

- I think bolding the "Figure X" in the figure caption helps set it off from the text - did we change this for a reason?
> I prefer bolding "Figure X"
> Also we should bold the first clause in the EDGI recommendations section as in the PDF.

- The link styling is different from last report, where we used underline. I think we should be consistent across the 100Days site, and if we're going to remove the underline, should do for all Parts, but should prob be a group decision, as I know we discussed this at length last time and group decision was to use underline. If we change to no underline, maybe we should match style of main site, with transition effect?
> I personally prefer the links without the underline, but I'm not sure if there was a styling discussion for the other reports. Maybe it's an accessibility concern?

- We need to add nav items for Data Methods & Data Visualization (Attached is a possible icon for the latter - haven't tried it in place but looks like it would match the others and not be too similar to Data Methods)
> I think the nav item for data looks good. We also need to **add one more nav item for quotes** about the report. Note that this isn't peer review - these are short 1-3 sentence quotes from various experts commenting on the final report.

- Should we make external links open in new tab, so we're not taking ppl away frmo the report? (as we do on main EDGI site)
> Yes please - I think that's a good idea (but of course, not the intra-report linking)

- Should we have nav element linking back to main EDGI site? It's pretty hard to find one's way to the main site from 100Days.
> I'm not against it

- Did we intentionally add the date to the top of this report? We didn't use in other reports, and I personally think it looks a bit better without (seems to make it feel instantly dated somehow)
> I'm ok with the date being there. Important for people to know what time period this refers to, especially if people will be referencing this months/years from now.

- We are still using different heading/subheading styling than we established early on for our report template and used in the previous report PDFs (I think we discussed this during discussion of prev web versions, but don't remember if it was resolved) do we want to be consistent w/ heading/subheading style or diverge online?
> I'm fine diverging online if it means less work. I'm more concerned about intra-document consistency 

- Can we match the section break in section "MISUSE OF LANGUAGE AND MISUNDERSTANDINGS" with that used in the PDF? Seems to stand out better when it's more spacious and uses more asterisks. Would be great to get same spacing above/below it, but I know sometimes spacing is tricky online.
> +1, I like the spacing and style in the PDF

- The comment boxes are different from PDF in that: a) heading style uses all caps, smaller font, and no horizontal rule, b) there's less margin above/below boxes and less padding within, and c) font color is black instead of charcoal grey. Were all those changes intentional?
> I support consistency here

- I think it'd be nice to add a little space at the end of the report before EDGI logo and make space above/below the HR consistent
> +1, also logo looks a little pixelated to my eye

- Should we increase the Footnotes/Comments headings to H2? They look a little small.
> +1 

- The image on the EPA Under Siege page isn't correctly scaling or isn't the correct image; should match the image on the index page
> pass

- Should we consider adjusting the Title/Subtitle size on individual report pages? The title seems really big and subtitle pretty small.
> Now that you mention it, I think that makes sense. But will defer to you on the exact ratio

- We still need to resolve question of using the cover image used for PDF vs the after-image in use now
> Yes, I've been thinking about this and am now leaning on just having the white line without the slider. I polled a few friends who don't know much about this work and they agree. Having the line makes it clear that there two images, but does not trick people into thinking it's a dynamic slider. The feedback I got is people will think we have a broken slider and will tell us about, which doesn't seem constructive. Will love to hear what others think...

**Comment by trinberg** [comment]
Adding a few more points:
- In the executive summary, can we bold "Key Findings", "These documented changes matter because they", "What are EDGI’s Recommendations?"
- In the "What are EDGI’s Recommendations?" section in the executive summary AND in the conclusion, the intro words for each rec should be bolded. "Transparency", "Responsible Web archiving", etc.
- Could we massage the Figure 1 caption so the URL doesn't split the caption and generate so much white space?

**Comment by suchthis** [comment]
Here are a couple icon ideas for the Quotes page you want to add @trinberg (cc @shaqsingh). The Comments icon already uses a quote box, so I think actual quotes are about all we can go with.

Flagging that we also still need to add those two new pages (Data Methods and Quotesd) to the web app, with proofed content.

**Comment by suchthis** [comment]
>Yes, I've been thinking about this and am now leaning on just having the white line without the slider. I polled a few friends who don't know much about this work and they agree. Having the line makes it clear that there two images, but does not trick people into thinking it's a dynamic slider. The feedback I got is people will think we have a broken slider and will tell us about, which doesn't seem constructive. Will love to hear what others think...

Ok, sounds good. Here's a version of the cover background with just the white line.

**Comment by trinberg** [comment]
I like the first quote icon (but think second one is also fine if others feel strongly)

Also, I'm compiling the **quotes** here: https://docs.google.com/document/d/13Q4-z0THVDvnXFOQzAE8s94W7OVOnWFcv4-TdLXixO4/edit - NOTE: I'm still compiling quotes and expecting 2-3 more.

Also, +1 on the cover image! The fact that the line clearly cuts off the green circles really makes it obvious there are 2 images.

**Comment by trinberg** [comment]
Draft methods write-up: https://docs.google.com/document/d/1KSgwdp0tjd3Q_tpnEppuNmLi4g3d56lOHiqEj41hj38/edit

File: assets/main.css [hunk 1]
```diff
@@ -215,6 +215,11 @@ section {
   background-position: bottom;
   background-size: cover;
 }
+#ch3bg {
+  background: url("/assets/partIII-cover-image.jpg");
```
Comment (dcwalk) [inline assets/main.css]: can we have this name match the other parts?

File: assets/post.css [hunk 2]
```diff
@@ -210,6 +214,12 @@ header {
   background-size: cover;
   background-repeat: no-repeat;
 }
+#bg3 {
+  background: url("/assets/partIII-cover-image.jpg");
```
Comment (dcwalk) [inline assets/post.css]: same RE: above, it seems like we follow "partN" the most (e.g. part3) so could we follow that as all the figure names are already in that pattern

File: changing-digital-climate.html [hunk 3]
```diff
@@ -0,0 +1,734 @@
+---
+layout: default
+title: "Part 3 - Changing the Digital Climate"
```
Comment (dcwalk) [inline changing-digital-climate.html]: Could we update this to follow the yaml frontmatter in the other sections?

File: changing-digital-climate.html [hunk 4]
```diff
@@ -0,0 +1,734 @@
...
+description: "How Climate Change Web Content is Being Censored Under the Trump Administration"
+author: "Toly Rinberg, Maya Anjur-Dietrich, Marcy Beck, Andrew Bergman, Justin Derry, Lindsey Dillon, Gretchen Gehrke, Rebecca Lave, Chris Sellers, Nick Shapiro, Anastasia Aizman, Dan Allan, Madelaine Britt, Raymond Cha, Janak Chadha, Morgan Currie, Sara Johns, Abby Klionsky, Stephanie Knutson, Katherine Kulik, Aaron Lemelin, Kevin Nguyen, Eric Nost, Kendra Ouellette, Lindsay Poirier, Sara Rubinow, Justin Schell, Lizz Ultee, Julia Upfal, Tyler Wedrosky, Jacob Wylie"
+og-image: "ch3-cover.jpg"
```
Comment (dcwalk) [inline changing-digital-climate.html]: See my point RE: filename conventions

File: changing-digital-climate.html [hunk 5]
```diff
@@ -0,0 +1,734 @@
...
+permalink: /changing-digital-climate/
+redirect_from:
```
Comment (dcwalk) [inline changing-digital-climate.html]: do we need to add an html redirect? (unsure what was publicized)

File: changing-digital-climate.html [hunk 6]
```diff
@@ -0,0 +1,734 @@
...
+comments: true
+---
+<body class="bg-white">
+    <div><a id="logo" href="/index.html" title="Home" class=""><img alt="edgi-logo" src="/assets/edgi-logo-b.png" style="width:5vh;min-width:35px; height:auto;"></a></div>
+    <header>
+        <div id="bg3" class="chbg"></div>
+        <div id="inner-wrapper">
+            <div id="heds">
+                <h2 class="f3 ttu tl">Part 3</h2>
+                <h1 id="p3_hed" class="f-6-l f4-m f4-ns ttu tl">CHANGING THE DIGITAL CLIMATE</h1>
```
Comment (dcwalk) [inline changing-digital-climate.html]: Can we use YAML variable here?

File: changing-digital-climate.html [hunk 7]
```diff
@@ -0,0 +1,734 @@
...
+                <h2 class="f3 fw3 ttu tl pv2">How Climate Change Web Content is Being Censored Under the Trump Administration</h2>
+            </div>
+            <div><p id="contrib" class="f5-l f7-m f7-ns fw1 lh-copy">Toly Rinberg, Maya Anjur-Dietrich, Marcy Beck, Andrew Bergman, Justin Derry, Lindsey Dillon, Gretchen Gehrke, Rebecca Lave, Chris Sellers, Nick Shapiro, Anastasia Aizman, Dan Allan, Madelaine Britt, Raymond Cha, Janak Chadha, Morgan Currie, Sara Johns, Abby Klionsky, Stephanie Knutson, Katherine Kulik, Aaron Lemelin, Kevin Nguyen, Eric Nost, Kendra Ouellette, Lindsay Poirier, Sara Rubinow, Justin Schell, Lizz Ultee, Julia Upfal, Tyler Wedrosky, Jacob Wylie</p></div>
```
Comment (dcwalk) [inline changing-digital-climate.html]: See point RE: YAML variables for this whole section

File: changing-digital-climate.html [hunk 8]
```diff
@@ -0,0 +1,734 @@
...
+        </div>
+    </header>
+    <div id="nav" class="w-100 pv1">
+            <a href="/index.html" title="Home" class="">
+                <img id="nav-logo" src="/assets/edgi-logo-b.png">
+            </a>
+            <div id="nav-txt" class="di"><h3 class="f5 ttu tl mb0 di">Part 3</h3><h3 class="f5 fw3 ttu tl mb0 di"> | Changing The Digital Climate</h3></div>
```
Comment (dcwalk) [inline changing-digital-climate.html]: Also can do the same here

File: changing-digital-climate.html [hunk 9]
```diff
@@ -0,0 +1,734 @@
...
+            <div class="back-to-top dib mh3 dn-ns dn-m dib-l">
+                <div class="dib">
+                    <a href="#h.bvng7vm3lrh2" class="dib tc">
+                        <img alt="top" src="/assets/top.png" class="" style="width:40px; margin-bottom:10px">
+                        <h3 class="f5 fw1 ttu">Back to Table of Contents</h3>
+                    </a>
+                </div>
+            </div>
+        </div>
+        <aside class="pt5"> <!--TODO-->
```
Comment (dcwalk) [inline changing-digital-climate.html:45]: Resolve this TODO?

File: changing-digital-climate.html [hunk 10]
```diff
@@ -0,0 +1,734 @@
...
+            <div class="aside-wrapper">
+                <a href="https://envirodatagov.org/wp-content/uploads/2017/09/Part-2-Pursuing-Toxic-Agenda.pdf" target="_blank" rel="noopener" target="_blank" rel="noopener" class="aside-el">
```
Comment (dcwalk) [inline changing-digital-climate.html]: Link to Part 2

File: changing-digital-climate.html [hunk 11]
```diff
@@ -0,0 +1,734 @@
+---
+layout: default
+title: "Part 3 - Changing the Digital Climate"
+description: "How Climate Change Web Content is Being Censored Under the Trump Administration"
+author: "Toly Rinberg, Maya Anjur-Dietrich, Marcy Beck, Andrew Bergman, Justin Derry, Lindsey Dillon, Gretchen Gehrke, Rebecca Lave, Chris Sellers, Nick Shapiro, Anastasia Aizman, Dan Allan, Madelaine Britt, Raymond Cha, Janak Chadha, Morgan Currie, Sara Johns, Abby Klionsky, Stephanie Knutson, Katherine Kulik, Aaron Lemelin, Kevin Nguyen, Eric Nost, Kendra Ouellette, Lindsay Poirier, Sara Rubinow, Justin Schell, Lizz Ultee, Julia Upfal, Tyler Wedrosky, Jacob Wylie"
```
Comment (dcwalk) [inline changing-digital-climate.html]: EDGI is listed as an author on previous report part

File: index.html [hunk 12]
```diff
@@ -22,7 +22,40 @@
...
     </div>
 </header>
+<section id="wrapper4" >
+    <div id="ch3bg" class="chbg"></div>
+    <div class="inner_wrapper">
+        <div class="part"><h2 class="f3 ttu">Part 3</h2></div>
+        <div class="ch_titles w-two-thirds-l w-100-m w-100-ns tl">
+            <a class="tl"href="/changing-digital-climate/">
+              <h1 class="f1-l f4-m f4-ns ttu tl">CHANGING THE DIGITAL CLIMATE</h1>
+              <!-- <h2 class="f3 fw3 ttu tl pv2">Environmental Injustice in the Early Trump Administration</h2> -->
```
Comment (dcwalk) [inline index.html]: Remove commented out section

**Review by dcwalk — COMMENTED** [review]
Nice! Left feedback inline

File: changing-digital-climate.html (hunk 3 above)
Comment (dcwalk) [reply changing-digital-climate.html]: Just gonna bump on this @shaqsingh as it'll be a requirement to have elements render correctly

File: changing-digital-climate.html (hunk 6 above)
Comment (dcwalk) [reply changing-digital-climate.html]: Also bump on this

File: changing-digital-climate.html (hunk 3 above)
Comment (shaqsingh) [reply changing-digital-climate.html]: sorry-- not super clear on how it differs currently

File: changing-digital-climate-data.html [hunk 13]
```diff
@@ -0,0 +1,87 @@
+---
+layout: default
+title: "Part 3 - EDGI Website Monitoring Methods"
```
Comment (dcwalk) [inline changing-digital-climate-data.html]: This needs to follow the format for YAML front matter

File: changing-digital-climate-quotes.html [hunk 14]
```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: "Part 3 - Changing the Digital Climate"
```
Comment (dcwalk) [inline changing-digital-climate-quotes.html]: Same with this one

**Comment by dcwalk** [comment]
Nice job @shaqsingh! 🎉 🎆 
I'm merging in, there are a couple of refinements we can make throughout the week

**Pull request merged**